src/dqc_us_0033_0036.py

```diff
@@ -93,7 +93,9 @@ def _get_default_dped(dped_facts):
     if len(keys) == 0:
         return None
     elif len(keys) == 1:
-        return [f for f in dped_facts.values()]
+        # Only one Document Period End Date, so that is our default
+        (_, value), = dped_facts.items()
```

Comment:
This is a standard Python construct around getting an item out of an iterator.

```dped_facts``` is a ```dict``` object.  The ```values()``` function returns an iterator (I believe a generated iterator) of type ```dict_items```.  This cannot be indexed into, but we know it must have one item (because of our ```len``` check leading into the ```if``` block).  Because of this, we can use the comma construct to only give us the first item (and in this case, only item) of the ```items()``` generator.